src/main/java/com/ericsson/ei/controller/AuthControllerImpl.java

```diff
@@ -74,7 +74,7 @@
     @ApiOperation(value = "To check backend status", response = String.class)
     public ResponseEntity<?> getCheckStatus() {
         try {
-            return new ResponseEntity<>("Backend is active", HttpStatus.OK);
+            return new ResponseEntity<>(HttpStatus.OK);
```

Comment:
I think you can can keep a message to the Http OK code, e.g: "Backend server is up and running"

Reply:
I tested it with frontend and somehow EI Status Button is red if REST API returns some message

Reply:
Ok, aren't we just checking the status code in frontend? ,,, then the message should be "don't care" value.

Reply:
Fixed. I have changed contentType for this ajax call in frontend

Reply:
Great